packages/slate-plugins/src/elements/mention/transforms/wrapMentionBrackets.ts

```diff
@@ -0,0 +1,63 @@
+import { Point, Range, Editor, Transforms } from "slate";
+
+const pointToStr = (point: Point | undefined) => {
+  if (!point) {
+    return "-";
+  }
+  let pointStr = "";
+  pointStr += point.path.join(".");
+  pointStr += " " + point.offset;
+  return pointStr;
+};
+
+const rangeToStr = (range: Range) => {
+  let rangeStr = "";
+  rangeStr += pointToStr(range.anchor);
+  rangeStr += " -> ";
+  rangeStr += pointToStr(range.focus);
+  return rangeStr;
+};
+
+export const wrapMentionBrackets = (editor: Editor, selection: Range) => {
+  console.dir(rangeToStr(selection));
+
+  const containingNode = Editor.node(editor, selection);
+  console.log("containing node:");
+  console.dir(containingNode);
+
```

Comment:
remove console logging

Reply:
intended to be in here for now, iteration is expensive and if something goes wrong i want to be able to quickly debug. will remove later. let me know if strong objections.